api.md

```diff
@@ -1832,29 +1832,75 @@ DROP MATERIALIZED VIEW contagg_view;
 
 ## refresh_continuous_aggregate() :community_function: [](refresh_continuous_aggregate)
 
-Refresh all buckets of a continuous aggregate between two points of
-time. 
+Refresh all buckets of a continuous aggregate in the _refresh window_
+given by `window_start` and `window_end`.
+
+A continuous aggregate materializes aggregates in time buckets (e.g.,
+min, max, average over 10 minutes of data), as determined by the
+continuous aggregate view query's `time_bucket` expression. Therefore,
+when refreshing the continuous aggregate, only buckets that completely
+fit within the refresh window will be refreshed. In other words, it is
+not possible to compute the aggregate over, e.g., half a bucket, and
+any buckets that do no fit within the given refresh window will be
+excluded.
+
+The function expects the window parameter values to have a time type
+that is compatible with the continuous aggregate's time bucket
+expression (e.g., if the time bucket is specified in `TIMESTAMP WITH
+TIME ZONE`, then the start and end time supplied should be a date or
+timestamp type). Note that a continuous aggregate using the `TIMESTAMP
+WITH TIME ZONE` type aligns with the UTC time zone, so any potential
+time zone shifts need to be accounted for when refreshing.
```

Comment:
I suggest to recast the sentence so that the comment in parentheses is not needed and can be, for example, a separate sentence. Remarks inside sentences, either using parentheses or em-dashes, have a tendency to make the sentence harder to read since the reader is detracted from following the main topic of the sentence.

Also, It is still not very clear if the fact that a type aligns with the UTC time zone means I can use it or not use it directly or what measures I need to take if I want to use it correctly.